src/components/OrderHistory/styles.ts

```diff
@@ -0,0 +1,98 @@
+// styles.ts
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const containerStyle = styled.div`
+  width: 250px; // This width might be too small if it's meant to contain the OrderDiv
+  height: 250px;
+  margin-right: auto; // Center the container if that's the intention
+  margin-left: auto; // Center the container if that's the intention
+  color: ${COLORS.black};
+  font-family: Public Sans;
+  font-size: 20px;
+  font-style: normal;
+  line-height: normal;
+`;
+
+export const imageGalleryStyle = {
+  display: 'flex',
+  overflowX: 'auto',
+  gap: '10px',
+  // Add more styles as needed
+};
+
+export const viewButtonStyle = {
+  marginTop: '10px',
+  backgroundColor: 'blue',
+  color: 'white',
+  padding: '10px 20px',
+  cursor: 'pointer',
+  // Add more styles as needed
+};
+
+export const ViewOrderButton = styled.button`
+  overflow: hidden;
+  color: var(--Black, #101010);
+  text-align: right;
+  text-overflow: ellipsis;
+  font-family: 'Public Sans', sans-serif;
```

Comment:
text will taken from teh file so you should be able to delete it here and just use the text compoentn